---
get_cloud/output_file: "pcd_images/aubo_i10/aubo_cloud_out.pcd"
save_output: true
translate_output: false

saving_target: true
saving_source: false
